### DISCUSSION TITLE
Is there any possibility to run Postfix (SMTP) and Dovecot (IMAP) on two separate FQDN with two different TLS certs?

### DISCUSSION THREAD

**Discussion by daks** [docker-mailserver/docker-mailserver#4252] [Q&A] [answered]
I imagine being able to
- run Postfix on smtp.example.com
- run Dovecot on imap.example.com
Each with its own TLS cert of course.

I imagine that it should be possible using a proxy but at the moment I don't use one (I couldn't find how to use Nginx for that and this is the reverse proxy I use), so  I wonder if it's possible to do it in another way.
Or if this feature could be added.

**Comment by polarathene** [comment, marked as answer]
Do you have a specific reason to need that? You could provision a cert with both SAN, or use a wildcard cert. No need for separate certificates, especially since they'd both be within the same container / server?

If you really did want to do such you could set the certificate paths for each yourself, but it's not something we specifically have support for as a convenience since separate certs doesn't add much value.

**Reply by daks** [reply]
In fact, I can have a cert with both SAN, I didn't thought about that. Thanks :)
